packages/core/core/constants.json

```diff
@@ -66,6 +68,7 @@
     "transactionMethods": {
       "eth_sendTransaction": "eth_sendTransaction",
       "eth_sendRawTransaction": "eth_sendRawTransaction",
+      "eth_sendRawPrivateTransaction": "eth_sendRawPrivateTransaction",
```

Comment:
In general, it doesn't feel quite right to me that plugin-specific constants find their way into `packages/core/core/constants.json`.

Could a plugin have it's own constants file in addition to using `embark-core/constants`?

Reply:
Good point. This constant was actually not being used in the plugin, so I removed it from core constants.